src/main/java/com/amazon/opendistroforelasticsearch/sql/exception/SQLFeatureDisabledException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.exception;
+
+public class SQLFeatureDisabledException extends Exception {
```

Comment:
I think this class represents a very concrete exception, so one should be able to use a default message, instead of always passing it.

Reply:
This has been already implemented like this internally, so its good to keep it like that. Do not want to divert much.

Reply:
I think this allows us to reuse this exception if we want to disable SQL feature for a different reason.

Reply:
np: I was suggesting to have a default constructor along with the one accepting a message, not critical though.